html/infrastructure/safe-passing-of-structured-data/shared-array-buffers/broadcastchannel-success.https.html

```diff
@@ -6,30 +6,27 @@
 <link rel="author" title="Domenic Denicola" href="mailto:d@domenic.me">
 <script src="/resources/testharness.js"></script>
 <script src="/resources/testharnessreport.js"></script>
+<script src="/common/utils.js"></script>
 
 <div id="log"></div>
 
 <script>
 "use strict";
 
 promise_test(t => {
+  const channelName = token();
```

Comment:
Why is this necessary? Is it guarding against running tests in multiple tabs at once? If so that'd be good to comment on, because for the purpose of our usual style of running a single test it doesn't seem necessary...

Reply:
One of your colleagues gave the feedback of preferring this over trying to avoid collisions manually, e.g., when adding more tests. Seems reasonable to me and also helps with running tests in parallel.

Reply:
I just don't understand why it's necessary at all in this case. There are not multiple tests here. If we are trying to support running multiple tests in parallel then we should note that explicitly since it's a rather unusual requirement.

Reply:
I was mostly just trying to make it easier for someone adding a new test. And reportedly this helps Safari and Servo.

Reply:
Seems reasonable, but not obvious from looking at the code, thus my suggestion to add a comment.

Reply:
In general the runner does support running multiple tests in parallel. Servo certainly uses this configuration in CI. Browsers with the concept of a profile typically use one profile per parallel instance, but e.g. Safari doesn't have the concept of profiles. By default we actually run all the tests in the same directory in the same parallel instance, but that's a customisable feature.